app/templates/courses/teacher-courses-view.jade

```diff
@@ -38,10 +38,17 @@ block content
                   | :
                   select.language-select.form-control
                     // TODO: Automate this list @scott
-                    option(value="python")
-                      | Python
-                    option(value="javascript")
-                      | JavaScript
+                    if course.get('slug') === 'web-dev-1'
+                      option(value="javascript")
+                        | HTML
```

Comment:
I'm not sure why this value is "javascript". Seems to work the same if this value is set to "html".